mrmustard/lab_dev/wires.py

```diff
@@ -300,6 +300,26 @@ def __bool__(self) -> bool:
         """
         return len(self.ids) > 0
 
+    def __eq__(self, other) -> bool:
+        r"""
+        Returns ``True`` if this ``Wires`` acts on the same modes as ``other``, ``False`` otherwise.
+        """
+        if self.output.bra.modes != other.output.bra.modes:
+            return False
+        if self.input.bra.modes != other.input.bra.modes:
+            return False
+        if self.output.ket.modes != other.output.ket.modes:
+            return False
+        if self.input.ket.modes != other.input.ket.modes:
+            return False
+        return True
+
+    def __neq__(self, other) -> bool:
```

Comment:
this is redundant (if a class has `__eq__`, python makes `__neq__` exactly like it's written here).